src/vault-bridge-tokens/vbETH/VbETH.sol

```diff
@@ -61,16 +66,14 @@ contract VbETH is VaultBridgeToken {
         require(mintedShares == shares, IncorrectAmountOfSharesMinted(mintedShares, shares));
     }
 
-    function _receiveUnderlyingToken(address, uint256 assets) internal override returns (uint256) {
+    function _receiveUnderlyingToken(address, uint256 assets) internal override {
```

Comment:
`safeTransferFrom` could also check the balance before/after like `VaultBridgeToken::_receiveUnderlyingToken()`

Reply:
Wdym?